### PR TITLE
PHP style guide updates for compatibility with PHP 7

### PR DESCRIPTION
**Pull request by cadie**
Corrects syntax errors and style guide conflicts such as: 
- Spaces following opening parenthesis and before closing parenthesis, before closing PHP tags, before and after if and else statements, etc
- Shorthand PHP tags
- Using `else if` instead of `elseif`
- Using lowercase `true` and `false`

Other adjustments:
- Typo corrections
- Updated instances of:
   ```
   $html = ob_get_contents(); ob_end_clean();
   return $html;
   ```
   to 
   `return ob_get_clean();`

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -141,59 +141,59 @@ function get_embed_html( $media_url ) {
  **/
 function gen_alerts_html()
 {
-		$alerts 		= get_posts(Array('post_type' => 'alert'));
-		$hidden_alerts	= Array();
+		$alerts 		= get_posts( array( 'post_type' => 'alert' ) );
```
Comment (cjg89) [inline functions.php]: Tabbing is a bit funky here

File: functions.php [hunk 2]
```diff
@@ -206,33 +206,38 @@ function gen_alerts_html()
...
 
-	if($size == 'photoset_photo') $size = 'full';
+	if ( $size == 'photoset_photo' ) $size = 'full';
 
-	$element_id		= (isset($options['element_id'])) ? $options['element_id'] : '';
-	$return_id		= (isset($options['return_id'])) ? $options['return_id'] : False;
-	$sent_attach	= (isset($options['sent_attach'])) ? $options['sent_attach'] : False;
+	$element_id		= ( isset( $options['element_id'] ) ) ? $options['element_id'] : '';
+	$return_id		= ( isset( $options['return_id'] ) ) ? $options['return_id'] : false;
+	$sent_attach	= ( isset( $options['sent_attach'] ) ) ? $options['sent_attach'] : false;
```
Comment (cjg89) [inline functions.php]: Tabbing

File: functions.php [hunk 3]
```diff
@@ -289,53 +293,60 @@ function get_img_html($post_id, $size = 'thumbnail', $options = Array())
...
 {
-	$home_page 	= (isset($atts['main_page'])) ? True : False;
-	$tag		= (isset($atts['tag'])) ? $atts['tag'] : False;
-	$category	= (isset($atts['category'])) ? $atts['category'] : False;
+	$home_page 	= ( isset( $atts['main_page'] ) ) ? true : false;
+	$tag		= ( isset( $atts['tag'] ) ) ? $atts['tag'] : false;
+	$category	= ( isset( $atts['category'] ) ) ? $atts['category'] : false;
 
-	$args = array_merge(Array(	'numberposts' => 1,
+	$args = array_merge( array(	'numberposts' => 1,
```
Comment (cjg89) [inline functions.php]: Tabbing on inner array is a little funky

**Review by rjucf — APPROVED** [review]

File: shortcodes.php [hunk 4]
```diff
@@ -607,23 +624,23 @@ function sc_ucf_video($atts = Array())
...
-	$args = Array('menu' => 'Resources', 'container' => '', 'menu_id'=> '');
+	$args = array( 'menu' => 'Resources', 'container' => '', 'menu_id'=> '' );
 
 	ob_start();
 	?>
-	<div class="<?=$css?>" id="resources">
+	<div class="<?php echo $css; ?>" id="resources">
 		<h2>Resources</h2>
-		<?=wp_nav_menu($args)?>
+		<?php echo wp_nav_menu( $args ); ?>
```
Comment (cjg89) [inline shortcodes.php]: `echo` isn't necessary when calling `wp_nav_menu()`; this function echo's its results by default

File: shortcodes.php [hunk 5]
```diff
@@ -820,34 +829,34 @@ function sc_subpage_features($atts = Array(), $id_only = False)
...
-	$css = (isset($atts['css'])) ? $atts['css'] : '';
+	$css = ( isset( $atts['css'] ) ) ? $atts['css'] : '';
 
-	$update = resolve_posts(	Array(	'tag' => $wp_query->queried_object->slug),
-								Array(	'post_type' => 'update',
-										'numberposts' => 1),
-								False,
-								False);
-	if($update !== False) {
+	$update = resolve_posts(	array(	'tag'         => $wp_query->queried_object->slug ),
```
Comment (cjg89) [inline shortcodes.php]: Funky tabbing

File: shortcodes.php [hunk 6]
```diff
@@ -863,9 +872,11 @@ function sc_external_stories( $atts = array() ) {
 	$linked_page_name = ( isset( $atts['linked_page_name'] ) ) ? $atts['linked_page_name'] : 'UCF in the News';
 	$show_description = ( isset( $atts['show_description'] ) ) ? filter_var( $atts['show_description'], FILTER_VALIDATE_BOOLEAN ) : false;
 
-	$stories = resolve_posts(	Array(	'tag' => $wp_query->queried_object->slug),
-								Array(	'post_type' => 'externalstory',
-										'numberposts' => $links_per_page ) );
+	$stories = resolve_posts(	array(	'tag' => $wp_query->queried_object->slug ),
```
Comment (cjg89) [inline shortcodes.php]: Tabs

File: shortcodes.php [hunk 7]
```diff
@@ -1295,41 +1294,45 @@ function sc_single_post_recommended($atts = Array())
...
 
 	ob_start();
-	foreach($experts as $expert) {
-		$stories = resolve_posts(	Array(),
-									Array(	'numberposts' => 5,
-											'exclude'     => Array($post->ID),
-											'tax_query'   => Array(
-																Array(	'taxonomy' => 'experts',
+	foreach ( $experts as $expert ) :
+		$stories = resolve_posts(	array(),
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: shortcodes.php [hunk 8]
```diff
@@ -1385,41 +1386,39 @@ function sc_expert_meta($atts = Array())
...
+	$css = ( isset( $atts['css'] ) ) ? $atts['css'] : '';
 
-	$term = get_term_by('name', get_post_meta($post->ID, 'expert_name', True), 'experts');
+	$term = get_term_by( 'name', get_post_meta( $post->ID, 'expert_name', true ), 'experts' );
 
-	$stories = resolve_posts(Array(),Array(	'numberposts' => 5,
-											'tax_query' => Array(
-																Array(
+	$stories = resolve_posts( array(),array( 'numberposts' => 5,
+											 'tax_query' => array(
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: shortcodes.php [hunk 9]
```diff
@@ -1428,44 +1427,42 @@ function sc_expert_tagged($atts = Array())
...
 
-	$term = get_term_by('name', get_post_meta($post->ID, 'expert_name', True), 'experts');
+	$term = get_term_by( 'name', get_post_meta( $post->ID, 'expert_name', true ), 'experts' );
 
-	$videos = resolve_posts(Array(),Array(	'post_type' => 'video',
+	$videos = resolve_posts(array(),array(	'post_type' => 'video',
 											'numberposts' => 5,
-											'tax_query' => Array(
-																Array(
+											'tax_query' => array(
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: shortcodes.php [hunk 10]
```diff
@@ -1474,18 +1471,17 @@ function sc_expert_videos($atts = Array())
...
 
-	$term = get_term_by('name', get_post_meta($post->ID, 'expert_name', True), 'experts');
+	$term = get_term_by( 'name', get_post_meta( $post->ID, 'expert_name', true ), 'experts' );
 
-	$photosets = resolve_posts(Array(),Array(	'post_type' => 'photoset',
+	$photosets = resolve_posts(array(),array(	'post_type' => 'photoset',
 												'numberposts' => 5,
-												'tax_query' => Array(
-																Array(
+												'tax_query' => array(
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: shortcodes.php [hunk 11]
```diff
@@ -1607,68 +1600,69 @@ function sc_photo_set($atts = Array())
...
+	$css = ( isset( $atts['css'] ) ) ? $atts['css'] : '';
 
-	$photo_sets = resolve_posts(Array(), Array(	'post_type' => 'photoset',
-												'numberposts' => -1));
-	$first = True;
-	ob_start();?>
-	<div id="photo-sets" class="<?=$css?>">
-	<?
+	$photo_sets = resolve_posts( array(), array( 'post_type' => 'photoset',
+												 'numberposts' => -1
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: shortcodes.php [hunk 12]
```diff
@@ -1761,33 +1764,37 @@ function sc_videos($atts = Array())
...
-	if(isset($atts['group'])) {
+	if ( isset( $atts['group'] ) ) {
 		$group_name = $atts['group'];
-		if( ($group = get_term_by('name', $group_name, 'groups')) !== False) {
-			$profiles = get_posts(Array('numberposts' => 4, 'post_type' => 'profile', 'group' => $group->slug));
-			ob_start();?>
-			<div id="profile-feature" class="<?=$css?>">
+		if ( ( $group = get_term_by( 'name', $group_name, 'groups' ) ) !== false ) {
+			$profiles = get_posts( array(	'numberposts' => 4,
+											'post_type' => 'profile',
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: shortcodes.php [hunk 13]
```diff
@@ -1826,53 +1833,56 @@ function sc_archive_articles($attrs) {
...
             <ul>
-        <?
-            for($i = 0; $i < count($articles);$i++) {
+        <?php
+            for ( $i = 0; $i < count( $articles ); $i++ ) {
                 $article = $articles[$i];
                 $class = '';
-                if($i == 0) {$class = 'first';}
-                if(($i + 1) == count($articles)) {$class = 'last';}
+                if ( $i == 0 ) {
```
Comment (cjg89) [inline shortcodes.php]: Tabbing

File: footer.php [hunk 14]
```diff
@@ -1,26 +1,26 @@
 			<div id="footer">
 				<div class="row">
 					<div class="span12 text-center">
-						<?=wp_nav_menu( array(
-							'theme_location' => 'social-links', 
+						<?php echo wp_nav_menu( array (
```
Comment (cjg89) [inline footer.php]: `echo` isn't necessary when calling `wp_nav_menu()`; this function echo's its results by default

File: footer-featured.php [hunk 15]
```diff
@@ -1,7 +1,7 @@
 			<div id="footer">
 				<div class="row">
 					<div class="span12 text-center">
-						<?=wp_nav_menu( array(
+						<?php echo wp_nav_menu( array (
```
Comment (cjg89) [inline footer-featured.php]: `echo` isn't necessary when calling `wp_nav_menu()`; this function echo's its results by default

**Review by cjg89 — APPROVED** [review]

**Review by jmbarne3 — APPROVED** [review]

**Review by jmbarne3 — APPROVED** [review]

**Pull request merged**